### PR TITLE
Disable ignored test.

### PR DESCRIPTION
**Pull request by nikhilnanivadekar**

### CODE REVIEW

File: unit-tests-java8/src/test/java/org/eclipse/collections/test/set/mutable/sorted/TreeSortedSetNoIteratorTest.java [hunk 1]
```diff
@@ -16,14 +16,11 @@
...
-import org.eclipse.collections.impl.test.junit.Java8Runner;
 import org.eclipse.collections.test.IterableTestCase;
 import org.eclipse.collections.test.NoIteratorTestCase;
-import org.junit.Ignore;
-import org.junit.runner.RunWith;
 
-@Ignore("Requires scapegoat tree implementation")
-@RunWith(Java8Runner.class)
+//@Ignore("Requires scapegoat tree implementation")
+//@RunWith(Java8Runner.class)
```
[the remark this answers is not in the document]
Comment (nikhilnanivadekar) [reply unit-tests-java8/src/test/java/org/eclipse/collections/test/set/mutable/sorted/TreeSortedSetNoIteratorTest.java]: There seems to be an issue in TC builds. The builds fail if the number of ignored tests is more than 10% than the previous build. However, getting the previous build is a problem as for every PR the state is maintained. This is causing the TC builds to erroneously fail. Once I comment the ignore and RunWith, this test is not detected as ignored. This is something which I need to report to TC but I need to have a passing and a failing build example.

**Comment by motlin** [comment]
I'm looking around in TC and I don't see any failed builds with an error message about number of ignored tests.

Anyway, we can fix that in TC configuration. No code change will be necessary.

**Comment by nikhilnanivadekar** [comment]
Here you go: 
https://teamcity.jetbrains.com/viewLog.html?buildId=1762647&tab=buildResultsDiv&buildTypeId=OpenSourceProjects_EclipseCollections_CompileTest

**Comment by motlin** [comment]
Do you have edit permissions?

I'm going to failure conditions:
https://teamcity.jetbrains.com/admin/editBuildFailureConditions.html?id=buildType:OpenSourceProjects_EclipseCollections_CompileTest

And I clicked disable on the one about ignored tests. These are just the auto-suggested failure conditions. They aren't necessary. It's implicit that an exit status code other than 0 also fails the build.

**Comment by nikhilnanivadekar** [comment]
ok makes sense. Closing this PR.

**Pull request closed without merging**